src/openrct2-ui/windows/About.cpp

```diff
@@ -218,37 +218,39 @@ static void window_about_openrct2_paint(rct_window* w, rct_drawpixelinfo* dpi)
 {
     window_about_openrct2_common_paint(w, dpi);
 
-    int32_t x, y, width;
+    int32_t width;
     rct_size16 logoSize;
 
     int32_t lineHeight = font_get_line_height(gCurrentFontSpriteBase);
 
-    x = w->windowPos.x + (w->width / 2);
-    y = w->windowPos.y + w->widgets[WIDX_PAGE_BACKGROUND].top + lineHeight;
+    ScreenCoordsXY coord(w->windowPos.x + (w->width / 2), w->windowPos.y + w->widgets[WIDX_PAGE_BACKGROUND].top + lineHeight);
```

Comment:
```suggestion
    ScreenCoordsXY aboutCoords (w->windowPos.x + (w->width / 2), w->windowPos.y + w->widgets[WIDX_PAGE_BACKGROUND].top + lineHeight);
```
Give a more meaningful name to the variable. This is just a suggestion, you might think of others.

This applies to other places in the diff too.